rgf/rgf.py

```diff
@@ -356,10 +344,13 @@ def fit(self, X, y):
         params.append("reg_depth=%s"%self.reg_depth)
         params.append("model_fn_prefix=%s"%os.path.join(loc_temp, self.file_prefix))
 
-        cmd = "%s train %s 2>&1"%(loc_exec, ",".join(params))
+        cmd = [loc_exec, "train", ",".join(params)]
 
         #train
-        output = platform_specific_Popen(cmd, stdout=subprocess.PIPE, shell=True).communicate()
+        output = subprocess.Popen(cmd,
+        	                      stdout=subprocess.PIPE,
+        	                      stderr=subprocess.STDOUT,
+        	                      universal_newlines=True).communicate()
```

Comment:
Tab character should not be used for indentation.
Your modification is correct.